app/controllers/price_groups_controller.rb

```diff
@@ -108,12 +105,30 @@ def destroy
 
   private
 
+  def price_group_members(search_term = nil)
+    unless SettingsHelper.feature_on?(:user_based_price_groups) && @price_group_ability.can?(:read, UserPriceGroupMember)
```

Comment:
This part of the logic feels more at home in a `before_action` to me - what do you think?  It's not really related to price group members